packages/find-root/src/index.ts

```diff
@@ -21,6 +22,7 @@ import {
  */
 const DEFAULT_TOOLS: Tool[] = [
   YarnTool,
+  BoltTool,
```

Comment:
would it work if we'd put `BoltTool` at the end?

Reply:
Yeah, it still works. I moved it down. It's still above `RootTool` as it seems like`RootTool` should always be last.

Anything else or are we good to merge?